### DISCUSSION TITLE
Fuxa x SQL

### DISCUSSION THREAD

**Discussion by PepeSM04** [frangoteam/FUXA#1188] [General]
Hello, how can I create an SQL database that can connect to fuxa?

**Comment by jbryan189** [comment]
Tinkering with it now, BUT have been successful just reading the device directly into a DuckDB or MariaDB using a Pi4.

**Comment by unocelli** [comment]
Hi, you can use odbc branch #1202

**Comment by flaritycat** [comment]
I've tried various options to connect to a MSSQL db via Fuxa on docker.
Both to a local SQL and to a non-local. 
The best I get, is the certificate error, and I've tried all the commands for auto accepting the certificate.
